src/main/java/com/amazonaws/encryptionsdk/internal/BouncyCastleConfiguration.java

```diff
@@ -0,0 +1,24 @@
+package com.amazonaws.encryptionsdk.internal;
+
+import org.bouncycastle.jce.provider.BouncyCastleProvider;
+
+import java.security.Security;
+
+/**
+ * This API is internal and subject to change
```

Comment:
Add some comments describing why this is necessary.

Reply:
Added comments